### PR TITLE
Issue 15 | Joss: initial draft

### LINKED ISSUE #15: Paper: prepare joss submission structure
Prepare submission structure for repository for JoSS https://joss.theoj.org/

Instructions https://joss.readthedocs.io/en/latest/submitting.html

### PR DESCRIPTION
**Pull request by IceKhan13**
### Summary

Works on JoSS paper submission

### Details and comments

Closes #15

### CODE REVIEW

**Comment by IceKhan13** [comment]
Compiled paper https://github.com/IceKhan13/purplecaffeine/suites/13002216069/artifacts/703896978

**Comment by IceKhan13** [comment]
[paper 2.pdf](https://github.com/IceKhan13/purplecaffeine/files/11519005/paper.2.pdf)

We still need more references. But this is a good first step

File: .github/workflows/draft-pdf.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Build JoSS paper
+
+on: [push]
```
Comment (mickahell) [inline .github/workflows/draft-pdf.yml]: Maybe not on every push, on every changes on the folder could do the job ?

```suggestion
on:
  workflow_dispatch:
  push:
    paths:
      - 'docs/paper/**'
```

**Review by mickahell — APPROVED** [review]
Nice draft :) just a suggestion to not having it running at every push

**Pull request merged**